### ISSUE TITLE
How do you e.g. exclude `kube-system` logs from ingestion?

### ISSUE THREAD

**Issue by philipn** [logdna/logdna-agent#37]
Apologies if this is more of a support question than a GitHub issue, but I haven't been able to figure out how to do something like the following:

  * Exclude logs if they're coming from a `kube-system` namespaced container, unless those logs are `ERROR` level.
  * Exclude App: syslog unless the log is `ERROR`

Figuring out how to search / filter based on this criteria is easy, but I'm wondering more about ingestion. Our very small K8s cluster is already very noisy :)

**Comment by leeliu** [comment]
You're correct, you can filter using LOGDNA_EXCLUDE broadly but not by line details as you mentioned. And yes, we process metadata after it reaches us (takes load off your systems). We generally encourage you to use Ingestion control (Gear icon > Ingestion) and you can pop it into the query box as `namespace:kube-system -level:error` and that should do it (i believe it comes in as `error` or `stderr`). You can even leave live tail on to see which ones got caught in the ingestion filter. (Give it 5 min to take effect).

**Comment by httran13** [comment]
Hello I dont see the "Ingestion" menu after clicking on gear icon, has this been updated?

**Comment by Selikoff** [comment]
Is there any way to exclude logs in the YAML file?  Aka, could you exclude all kube-system logs via the YAML script?
